yantra/manager.py

```diff
@@ -39,8 +39,18 @@ def _get_modules(self):
         """Discover all modules in the path"""
         plugin_modules = []
 
+        if isinstance(self.plugin_type.path, list):
+           for p in self.plugin_type.path:
+               plugin_modules = self._get_modules_in_path(plugin_modules, p)
```

Comment:
I think we should be appending to the list here

Reply:
We might also want to make it a set to remove any potential duplicate modules

Reply:
Good idea, I'll keep this in mind if/when I re-open the MR. Thanks @RouganStriker  nice catch